docs/ossm/ossm2-migration/cert-manager/README.md

```diff
@@ -0,0 +1,303 @@
+reviews-v3-596954cfd6-jnb6n.bookinfo         Kubernetes     SYNCED     SYNCED     SYNCED     SYNCED              istiod-ossm3-v1-24-1-d5b9b4c89-ccz4v     1.24.1
+```
+ -->
+
+Starting with a `ServiceMeshControlPlane` with cert-manager configured:
+
+```yaml
+apiVersion: maistra.io/v2
+kind: ServiceMeshControlPlane
+metadata:
+  name: basic
+  namespace: istio-system
+spec:
+  ...
+  security:
+    certificateAuthority:
+      cert-manager:
+        address: cert-manager-istio-csr.cert-manager.svc:443
+      type: cert-manager
+    dataPlane:
+      mtls: true
+    identity:
+      type: ThirdParty
+    manageNetworkPolicy: false
+```
+
+You will need to perform these updates to your istio-csr deployment:
+
+- The `app.istio.revisions` field needs to include your 3.0 control plane revision _before_ you create your `Istio` resource.
+
+  Adding your 3.0 control plane revision to your istio-csr deployment will ensure that proxies can properly communicate with the 3.0 control plane.
+
+  ```console
+  helm upgrade cert-manager-istio-csr jetstack/cert-manager-istio-csr \
+      --install \
+      --reuse-values \
+      --namespace cert-manager \
+      --wait \
+      --set "app.istio.revisions={basic,ossm3-v1-24-1}"
+  ```
+
+  Depending on whether you will use a `RevisionBased` update strategy or an `InPlace` update strategy, your revision name will vary. If using an `InPlace` strategy, your revision name will match your `Istio` name. If using a `RevisionBased` strategy, revision names use the following format, `<istio-name>-v<major_version>-<minor_version>-<patch_version>`. For example: `ossm3-v1-24-1`.
+
+- The `app.controller.configmapNamespaceSelector` field needs to be either unset _before_ the migration begins or updated _after_ you have completed your migration.
+
+  If you have set the `app.controller.configmapNamespaceSelector` field on your istio-csr deployment to `maistra.io/member-of`, you will need to update this accordingly. If you haven't set this field, you can keep it unset.
+
+  If unset the `configmapNamespaceSelector` field on your istio-csr deployment, the istio CA configmap will be injected into every namespace. `MultiTenant` deployments with more than one `ServiceMeshControlPlane` in the cluster should not remove this field since the wrong CA configmap would likely get written to the namespace. If you are keeping the field set, you need to wait until **after** you have completed your migration to update the `configmapNamespaceSelector` field. Otherwise namespaces without the injection label will no longer have the configmap CA injected.
+
+  To unset this field:
+
+  ```console
+  helm upgrade cert-manager-istio-csr jetstack/cert-manager-istio-csr \
+      --install \
+      --reuse-values \
+      --namespace cert-manager \
+      --wait \
+      --set "app.controller.configmapNamespaceSelector=null"
```

Comment:
This was not working for me, the value was unchanged after this. I used `--set "app.controller.configmapNamespaceSelector="`.

Reply:
I must've copied an array example that used `null`